packages/@aws-cdk/aws-route53/lib/hosted-zone.ts

```diff
@@ -190,6 +191,14 @@ export interface PublicHostedZoneProps extends CommonHostedZoneProps {
    * @default false
    */
   readonly caaAmazon?: boolean;
+
+  /**
+   * The principle for which a assumed by role is created
```

Comment:
Maybe something closer to this?

```suggestion
   * A principal which is trusted to assume a role for zone delegation
```